docs/content/preview/api/ysql/exprs/func_setval.md

```diff
@@ -0,0 +1,93 @@
+---
+title: setval() function [YSQL]
+headerTitle: setval()
+linkTitle: setval()
+description: Set and return the value for the specified sequence.
+menu:
+  preview:
+    identifier: api-ysql-exprs-setval
+    parent: api-ysql-exprs
+isTocNested: true
+showAsideToc: true
+---
+
+
+## Synopsis
+
+Use the `setval( sequence_name , value, is_called)` function to set and return the value for the specified sequence.
+`UPDATE` privilege on the sequence is required to call this function.
+
+Calling the function with two parameters, will default `is_called` to true, meaning, the `nextval` will advance the sequence 
+prior to returning the value and `currval` will also return the specified value.
+
+When called with `is_called` as `false`, `nextval` will return the specified value and the value reported by `currval` will not be changed.
+
+`setval()` returns just the value of it's second argument.
+
+## Semantics
+
+### _sequence_name_
+
+Specify the name of the sequence.
+
+### _value_
+
+Specify the value of the sequence.
+
+### _is_called_
+
+Specify `is_called` to `true` or `false`.
```

Comment:
"Specify `is_called` to `true` or `false`."  >

Set `is_called` to `true` or `false`.